test/Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests/Common/Command/HttpCommand.cs

```diff
@@ -0,0 +1,94 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Extensions.Logging;
+using Polly;
+using Polly.Retry;
+using System;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.Common
+{
+	// Responsible for creation and execution of commands to handle http request response.
+	public class HttpCommand : IInfraCommand<HttpResponseMessage>
```

Comment:
HttpCommand is a normal command which is not extending any functionality of infra commanding, so it will be great if we should extending this with ICommand Interface and refactor the IInfraCommand interface to extend the ICommand functionality